server/services/supportedPrisonsService.ts

```diff
@@ -40,14 +41,10 @@ export default class SupportedPrisonsService {
     return orchestrationApiClient.getSupportedPrisonIds()
   }
 
-  async getPrisonConfig(
-    username: string,
-    prisonCode: string,
-  ): Promise<{ maxTotalVisitors: number; policyNoticeDaysMin: number }> {
+  async getPrison(username: string, prisonId: string): Promise<Prison> {
```

Comment:
This is the key part of the change: get whole `Prison` rather than just some of the config values.